src/libraries/EventsLib.sol

```diff
@@ -63,6 +63,15 @@ library EventsLib {
     /// @notice Emitted when the `withdrawQueue` is set to `newWithdrawQueue`.
     event SetWithdrawQueue(address indexed allocator, Id[] newWithdrawQueue);
 
+    /// @notice Emitted when a reallocation supplies `supplied` assets to the market identified by `id`.
+    event ReallocateSupply(Id id, uint256 supplied);
+
+    /// @notice Emitted when a reallocation withdraws `withdrawn` assets from the market identified by `id`.
+    event ReallocateWithdraw(Id id, uint256 withdrawn);
+
+    /// @notice Emitted when a reallocation added or removed assets from idle.
+    event ReallocateIdle(uint256 added, uint256 removed);
```

Comment:
```suggestion
    event ReallocateIdle(uint256 idle);
```

Isn't it simpler?

Reply:
Or we could have to events :` ReallocateIncreaseIdle` and  `ReallocateDecreaseIdle`

Reply:
It is really easy to know if this is increasing or decreasing the idle amount in fact, so IMO the @Rubilmax suggestion is ok